### PR TITLE
Added contacts list UI

### PR DESCRIPTION
**Pull request by soorajjtk**

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import Menubar from "components/Common/Menubar";
+import Table from "components/Dashboard/Contacts/Table";
+
+import { CONTACTS_DATA, MENUBAR_DATA } from "./constants";
+
+function Contacts() {
+  const [searchTerm, setSearchTerm] = useState("");
+  const [, setCategory] = useState("All");
+  const [, setShowNewContactPane] = useState(false);
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx:14]: Not sure I understand the need to make this a state if we aren't actually utilising this as a state. Why are these declared as states?

**Review by yedhink — COMMENTED** [review]
@soorajjtk _a

**Comment by yedhink** [comment]
@soorajjtk What's the reason for assigning me this PR again when I have already requested some review changes from your end? 

image

File: app/javascript/src/components/Dashboard/Contacts/index.jsx (hunk 1 above)
Comment (soorajjtk) [reply app/javascript/src/components/Dashboard/Contacts/index.jsx:14]: showNewContactPane is passed in CreatePane component which is added in next pull request.
Incase of category I forgot to add it to the menubar title. I have created a new pull request with that change.
https://github.com/soorajjtk/neeto-ui-challenge/pull/12

**Comment by yedhink** [comment]
@soorajjtk You should start acknowledging questions! Read: https://handbook.neetokb.com/articles/acknowledging-a-message

image

**Comment by soorajjtk** [comment]
I had assigned since I was unable to assign it as mentioned in neeto UI challenge yesterday. Now seeing you have already reviewed I have unassigned you. I will take care of it going ahead.

**Pull request merged**